dev-libs/libunistring/libunistring-0.9.6.recipe

```diff
@@ -2,35 +2,44 @@ SUMMARY="A library for manipulating Unicode strings"
 DESCRIPTION="libunistring provides functions for manipulating Unicode strings and for \
 manipulating C strings according to the Unicode standard."
 HOMEPAGE="http://www.gnu.org/software/libunistring/"
-SOURCE_URI="http://ftp.gnu.org/gnu/libunistring/libunistring-$portVersion.tar.gz"
-LICENSE="GNU LGPL v2.1"
 COPYRIGHT="1998-2014 Free Software Fundation, Inc."
-CHECKSUM_SHA256="9625eec2507f4789ebb6fc48ebda98be0e0168979a2f68aa8b680bf8eeabbd47"
+LICENSE="GNU LGPL v2.1"
 REVISION="1"
+SOURCE_URI="http://ftp.gnu.org/gnu/libunistring/libunistring-$portVersion.tar.gz"
```

Comment:
You might wish to use their geographic redirector service to one of their mirrors:
```
SOURCE_URI="http://ftpmirror.gnu.org/libunistring/libunistring-$portVersion.tar.gz"
```